far/vmenu.cpp

```diff
@@ -172,6 +171,45 @@ int find_nearest(std::ranges::contiguous_range auto const& Range, const int Pos,
 			: FindPos(drop(Pos), take(Pos) | reverse));
 }
 
+std::pair<int, int> Intersect(std::pair<int, int> A, std::pair<int, int> B)
+{
+	assert(A.first < A.second);
+	assert(B.first < B.second);
+
+	if (B.first < A.first)
+		std::swap(A, B);
```

Comment:
Let's use ranges::swap for consistency.
Same below.

Reply:
OK. Sorry, below where?

Reply:
Yes